test/CMakeLists.txt

```diff
@@ -208,10 +214,11 @@ function(add_custom_test NAME)
     # Many custom tests do test only FP32 data type and therefore
     # added only if none of MIOPEN_TEST_HALF, MIOPEN_TEST_INT8, MIOPEN_TEST_BFLOAT16
     # are set, except the test is allowed explicitly.
-    if((NOT (MIOPEN_TEST_INT8 OR MIOPEN_TEST_BFLOAT16 OR MIOPEN_TEST_HALF))
+    if((NOT (MIOPEN_TEST_INT8 OR MIOPEN_TEST_BFLOAT16 OR MIOPEN_TEST_HALF OR MIOPEN_TEST_MIOTENSILE))
     OR (MIOPEN_TEST_BFLOAT16 AND ${PARSE_ALLOW_BFLOAT16})
     OR (MIOPEN_TEST_HALF AND ${PARSE_ALLOW_HALF})
-    OR (MIOPEN_TEST_INT8 AND ${PARSE_ALLOW_INT8}))
+    OR (MIOPEN_TEST_INT8 AND ${PARSE_ALLOW_INT8})
+    OR (NOT (MIOPEN_TEST_MIOTENSILE AND (NAME IN_LIST SKIP_TESTS))))
```

Comment:
@ce1adon It should be
```
OR (MIOPEN_TEST_MIOTENSILE AND NOT (NAME IN_LIST SKIP_TESTS)))
```
Current code enables ALL tests when MIOPEN_TEST_MIOTENSILE=off, thus affecting non-miopentensile tests (half etc). I will fix that.

Reply:
This is under the condition of (NOT (MIOPEN_TEST_INT8 OR MIOPEN_TEST_BFLOAT16 OR MIOPEN_TEST_HALF OR MIOPEN_TEST_MIOTENSILE)), which means this only applies to fp32 cases. Do you mean fp32 doesn't need to test all the cases?

Reply:
No, this is not "under" that confition. It is just one of ORed conditions. Alternate formatting:
```
if(
    (NOT (MIOPEN_TEST_INT8 OR MIOPEN_TEST_BFLOAT16 OR MIOPEN_TEST_HALF OR MIOPEN_TEST_MIOTENSILE)) // (1)
    OR (MIOPEN_TEST_BFLOAT16 AND ${PARSE_ALLOW_BFLOAT16}) // (2)
    OR ( MIOPEN_TEST_HALF AND ${PARSE_ALLOW_HALF}) // (3)
    OR (MIOPEN_TEST_INT8 AND ${PARSE_ALLOW_INT8}) // (4)
    OR (NOT (MIOPEN_TEST_MIOTENSILE AND (NAME IN_LIST SKIP_TESTS))) // (5)
) ...
```
When MIOPEN_TEST_MIOTENSILE=Off, condition (5) is always true. This works properly for MIOpenTensile, that's why you've missed that, I guess.

Reply:
Fixed in 318f48073e1bee6d1 (not yet merged to develop)

Reply:
Fixed in #750